File: LaborMarket.Api/Models/EmployerModel.cs
using System.ComponentModel.DataAnnotations;

namespace LaborMarket.Api.Models
{
	public class EmployerModel
	{
		[Key]
		public int EmployerId { get; set; }
		public string CompanyName { get; set; } = null!;
		public string ContactEmail { get; set; } = null!;
		public string ContactPhone { get; set; } = null!;
	}
}

File: LaborMarket.Web/src/app/models/employer-model.ts
export type EmployerDataModel = {
  employerId: number;
  companyName: string;
  contactEmail: string;
  contactPhone: string;
  description: string;
  profileImageUrl: string;
};

export type CreateEmployerModel = {
  companyName: string;
  contactEmail: string;
  contactPhone: string;
  password: string;
  role: string;
};
